Gruntfile.js

```diff
@@ -1,7 +1,7 @@
 /* jshint node:true */
 module.exports = function( grunt ) {
 	'use strict';
-	const sass = require( 'node-sass' );
+	var sass = require( 'node-sass' );
```

Comment:
The grunt build was failing, I think a js linter was complaining about this:

```
  Gruntfile.js
      4 |    const sass = require( 'node-sass' );
             ^ 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).
```

Since there's no other es6 in this file, or in the rest of the code base (I think!), I thought it was more consistent to keep this simple. I believe it's a recent change, maybe others are working around it differently, or this is fixed upstream now? (I asked about how to address this in proton slack and mentioned my workaround.)

Happy to pull this from this PR, as it's not relevant to what I'm doing. I'm still looking for assistance getting this built/minified correctly though.